.ci/common.sh

```diff
@@ -48,7 +48,15 @@ check_core_dumps() {
 }
 
 setup_deps() {
+	sudo pip --upgrade pip
 	sudo pip install neovim
+
+	# For pip3
+	# https://github.com/travis-ci/travis-ci/issues/1528 
+	curl http://python-distribute.org/distribute_setup.py | sudo python3
```

Comment:
This would be a good time to add `set -o errexit` to the top of the script. 

See http://robertmuth.blogspot.com/2012/08/better-bash-scripting-in-15-minutes.html?_sm_au_=irVqqt1nVDrrsQ32